Frameworks/UIKit/UIScrollView.mm

```diff
@@ -894,7 +894,11 @@ - (void)_refreshOrigin {
     [self setNeedsLayout];
 
     if ([self.delegate respondsToSelector:@selector(scrollViewDidScroll:)]) {
-        [self.delegate scrollViewDidScroll:self];
+        // NOTE: We have to push this off of the ScrollViewer's callback thread to work around
+        // issue #1865.
+        dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
>dispatch_async [](start = 8, length = 14)

my only concern is that we probably should dispatch_async all delegate calls if we do this, otherwise, the sequence between this and those which used to fire after this might be changed. But This might or might not, unfortunately we don't have enough apps to tell.  I 'd prefer to work-around this on app given this is xaml framework issue based on commucation. 

Thoughts?